tasks/tasks.go

```diff
@@ -33,6 +33,24 @@ const (
 	NOTFOUND
 )
 
+var DefaultTaskExecuter = NewTaskExectuer()
```

Comment:
@sandlerben looking in atom now, 
<img width="404" alt="screen shot 2016-02-29 at 12 42 16 am" src="https://cloud.githubusercontent.com/assets/4250521/13386700/4ebffe5a-de7d-11e5-8b71-a853d49a08f1.png">

Can you add comment please?